### PR TITLE
Javascript object Methods .defineProperties() 

### LINKED ISSUE #4025: [Term Entry] JavaScript Objects .defineProperties()
### Reason/inspiration (optional)

We would like a new term entry on the `Object.defineProperties()` method for [Objects](https://www.codecademy.com/resources/docs/javascript/objects): defineProperties. The entry should go in a new file under `docs/content/javascript/concepts/objects/terms/defineProperties/defineProperties.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```js `code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by Brag2gr8**
### Description

Added new entry for Javascript Object Methods defineProperties()
Entry Location: content/javascript/concepts/objects/terms/defineProperties/defineProperties.md. closes #4025

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/objects/terms/defineProperties/defineProperties.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  - 'Properties'
+  - 'Objects'
+  - 'Functions'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `Object.defineProperties()` is used to define new or modify existing properties directly on an object or to add multiple properties to an object at once. This method is often used for property descriptors and fine-grained control over object properties.
```
Comment (ishg-153) [inline content/javascript/concepts/objects/terms/defineProperties/defineProperties.md:17]:
```suggestion
The **`.defineProperties()`** method is used to define new or modify existing properties directly on an object or to add multiple properties to an object at once. This method is often used for property descriptors and fine-grained control over object properties.
```

File: content/javascript/concepts/objects/terms/defineProperties/defineProperties.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+## Syntax
+
+```pseudo
+Object.defineProperties(object, descriptors)
+```
+
+- `object`: The object on which to define or modify it's properties.
+- `descriptors`: An object where each key represents a property name, and the corresponding value is the property descriptor.
```
Comment (ishg-153) [inline content/javascript/concepts/objects/terms/defineProperties/defineProperties.md:26]:
```suggestion
- `descriptors`: An object where each key represents a property name, and the corresponding value is the property descriptor.

[optional keys of descriptors]
```
`descriptors` has optional keys `value` and `writable` that you've used in the example below. Write about these keys in points. There are also keys `get` and `set` if I'm not wrong.

File: content/javascript/concepts/objects/terms/defineProperties/defineProperties.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+## Example
+
+In this example `Object.defineProperties` is used to define two properties (`property1` and `property2`) on the `myObject` object. Each property has a descriptor object specifying attributes like `value` and `writable`.
+The `value` attribute is the value of the property and the `writable` attribute determines whether the property can be modified.
```
Comment (ishg-153) [inline content/javascript/concepts/objects/terms/defineProperties/defineProperties.md:31]:
```suggestion
```
This sentence won't be necessary as we are adding that in the syntax section now.

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@Brag2gr8 This is a great entry overall. I have left suggestions, please address those. Let me know when you are done.

**Comment by Brag2gr8** [comment]
@ishg-153 I just updated the changes

**Comment by dakshdeepHERE** [comment]
I did some minor changes and this LGTM 🚀

**Comment by dakshdeepHERE** [comment]
@Brag2gr8 can you change the current Branch name a bit? Because of the branch name ending with `-` I'm not able to push the changes I made for the yarn checks here. Make sure you keep the branch name simple and it doesn't end with `()` or `-`.

**Pull request closed without merging**